src/openMVG/tracks/tracks.hpp

```diff
@@ -71,7 +78,14 @@ struct TracksBuilder
   {
     // 1. We need to know how much single set we will have.
     //   i.e each set is made of a tuple : (imageIndex, featureIndex)
-    std::set<indexedFeaturePair> allFeatures;
+    std::unordered_set<indexedFeaturePair, image_feature_hash> allFeatures;
```

Comment:
Is it possible to have two similar `indexedFeaturePair` at all?

Reply:
We should not since we are using a set